crates/ruff_python_formatter/src/expression/string.rs

```diff
@@ -207,7 +207,11 @@ impl Format<PyFormatContext<'_>> for FormatStringPart {
 
         write!(f, [prefix, preferred_quotes])?;
 
-        let (normalized, contains_newlines) = normalize_string(raw_content, preferred_quotes);
+        let (normalized, contains_newlines) = normalize_string(
+            raw_content,
+            preferred_quotes,
+            matches!(prefix, StringPrefix::RAW | StringPrefix::RAW_UPPER),
```

Comment:
Nit: I would implement this as a method on `StringPrefix` to reduce the complexity here
```suggestion
			prefix.is_raw_string()
```

Reply:
Sounds good. I'll make the change.